aiaccel/scheduler/abstract_scheduler.py

```diff
@@ -22,13 +22,15 @@ class AbstractScheduler(AbstractModule):
             command or qstat command.
     """
 
-    def __init__(self, options: dict) -> None:
+    def __init__(self, options: dict, Model: Any = None) -> None:
```

Comment:
`Model: Any` でなく `typing.Type` じゃないでしょうか．

Reply:
```suggestion
    def __init__(self, options: dict, Model: Any) -> None:
```
ここ `None` が代入されると，`self.scheduler.Model()` したら落ちますね？
例外処理もないので，ここは必ず引数を指定するようにして下さい．

Reply:
坂東さん、荒本さん、レビューなど、ありがとうございます。

Typeに変更しました。

Noneで落ちます。その場合は、バグでtestのtracebackなどで確認できます。
後程、補足させていただきます。

Reply:
コメントありがとうございます．
`self.Model` にアクセスして `None` が返ってくるというのは，他のモジュールにSchedulerのメンバである `self.Model` が `None` でないかをチェックさせることを強要しており，実装として変に思います

`self.scheduler.Model` はJobで使われてるので，直接 `self.scheduler.Model` というメンバにアクセスさせるのでなく，__init__にModelのインスタンスを指定させる方が良い気がします．

また，明日の会議で議論させてください．

よろしくお願いします．